### PR TITLE
Change Favourites Page layout

### LINKED ISSUE #1555: Favourites page - remove right side content, add Print button (not share)
Design: https://app.zeplin.io/project/5acbd659dc553ec84805afd4/screen/5c006921d385f82c6d6bb073

screen shot 2018-12-07 at 8 54 49 am

### PR DESCRIPTION
**Pull request by brdunfield**
Closes #1555

### CODE REVIEW

**Comment by brdunfield** [comment]
Also implemented #1554

**Comment by brdunfield** [comment]
[Users understand the next steps they need to take to apply](https://trello.com/c/0VQPiAtf/112-users-understand-the-next-steps-they-need-to-take-to-apply)

**Comment by courtneycl** [comment]
I think this is picking up the old `<h1>` or something  - it's 52 px when should be 36 px. 

image

File: components/favourites.js [hunk 1]
```diff
@@ -97,42 +86,56 @@ export class Favourites extends Component {
...
+                      }
+                    />
+                  ) : null}
+                </Grid>
+              </Grid>
 
               <Grid container spacing={24}>
+                <Header
+                  className={"BenefitsCounter"}
+                  size="xl"
```
Comment (SupeDeDupe) [inline components/favourites.js]: `size="lg"` for 36px font.

File: components/favourites.js [hunk 2]
```diff
@@ -97,42 +86,56 @@ export class Favourites extends Component {
...
+                  headingLevel="h1"
+                >
+                  {t("favourites.saved_benefits", {
+                    x: filteredBenefits.length
+                  })}
+                </Header>
```
Comment (SupeDeDupe) [inline components/favourites.js:142]: `<Body>{t("B3.check eligibility")}</Body>` to add the "Please read the eligibility criteria" text under the heading.

**Review by SupeDeDupe — COMMENTED** [review]
Good job 👍 just a few small changes...

**Review by SupeDeDupe — APPROVED** [review]
Nice! 🐯

**Pull request merged**